OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -45,7 +45,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x86'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>..\</OutputPath>
+    <OutputPath>bin\Release</OutputPath>
```

Comment:
Was changing these paths intentional? This breaks at runtime since make no longer dumps dependencies into the right places.